src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -174,18 +173,38 @@
       + " indices at startup. This is useful when the indices are a direct mapping "
       + " of the Kafka topics.";
 
+  private static final String ELASTICSEARCH_SECURITY_PROTOCOL_CONFIG = "elastic.security.protocol";
+  private static final String ELASTICSEARCH_SECURITY_PROTOCOL_DOC =
+      "The security protocol to use when connecting to Elasticsearch. "
+          + "Values can be `PLAINTEXT` or `SSL`.";
```

Comment:
Since the name prefix here doesn't match the other elastic config names ("elastic.https"), can we get some additional text specifying that this will determine whether those other configs are required or ignored?

Reply:
Thanks, added a clarification.